ocpp/v16/charging_profile.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from ocpp.v16.enums import (
+    ChargingProfileKindType,
+    ChargingProfilePurposeType,
+    ChargingRateUnitType,
+    RecurrencyKind,
+)
+
+
+@dataclass
+class ChargingSchedulePeriod:
+    """Charging schedule period structure defines a time period
+    in a charging schedule, as used in: ChargingSchedule."""
+
+    start_period: int
+    limit: float
+    number_phases: Optional[int] = None
+
+
+@dataclass
+class ChargingSchedule:
+    """Charging schedule structure defines a list of charging periods,
+    as used in: GetCompositeSchedule.conf and ChargingProfile."""
+
+    charging_rate_unit: ChargingRateUnitType
+    charging_schedule_period: List[ChargingSchedulePeriod]
+    duration: Optional[int] = None
+    start_schedule: Optional[str] = None
+    min_charging_rate: Optional[float] = None
+
+
+@dataclass
+class ChargingProfile:
```

Comment:
This datatype already exists in 
https://github.com/mobilityhouse/ocpp/blob/a0b4820e4ae9c4758d520d35136f014104c5b184/ocpp/v16/datatypes.py#L62

What is the rationale for adding it here again?